src/schema/facet.rs

```diff
@@ -122,6 +122,11 @@ impl Facet {
     pub fn to_path(&self) -> Vec<&str> {
         self.encoded_str().split(|c| c == FACET_SEP_CHAR).collect()
     }
+
+    /// Extract the path as a string from the `Facet`.
+    pub fn to_path_string(&self) -> String {
+        format!("/{}", self.to_string())
```

Comment:
The initial double slash is not a bad idea, but this is not the way `Facet` have been designed.

We want `.to_path_string()` to be the opposite of `Facet::from(&str)`.

Reply:
My bad, was due to the first comment you made on this issue. Can see what the original issue said now.